datahub-web-react/src/app/ingest/secret/SecretBuilderModal.tsx

```diff
@@ -53,48 +57,82 @@ export const SecretBuilderModal = ({ initialState, visible, onSubmit, onCancel }
+                                message: 'Enter a name.',
+                            },
+                            { whitespace: true },
+                            { min: 1, max: 50 },
+                            {
+                                pattern: SECRET_NAME_ALPHANUMERIC,
+                                message: '',
+                            },
+                        ]}
+                        hasFeedback
+                    >
+                        <Input
+                            placeholder="A name for your secret"
+                            value={secretBuilderState.name}
+                            onChange={(event) => setName(event.target.value)}
+                        />
+                    </Form.Item>
                 </Form.Item>
-                <Form.Item name="value" label={<Typography.Text strong>Value</Typography.Text>}>
+                <Form.Item label={<Typography.Text strong>Value</Typography.Text>}>
                     <Typography.Paragraph>
                         The value of your secret, which will be encrypted and stored securely within DataHub.
                     </Typography.Paragraph>
-                    <Input
-                        placeholder="The value of your secret"
-                        value={secretBuilderState.value}
-                        onChange={(event) => setValue(event.target.value)}
-                    />
+                    <Form.Item
+                        name="value"
+                        rules={[
+                            {
+                                required: true,
+                                message: 'Enter a value.',
+                            },
+                            // { whitespace: true },
+                            { min: 1 },
+                        ]}
+                        hasFeedback
+                    >
+                        <Input
+                            placeholder="The value of your secret"
+                            value={secretBuilderState.value}
+                            onChange={(event) => setValue(event.target.value)}
+                        />
+                    </Form.Item>
                 </Form.Item>
-                <Form.Item name="value" label={<Typography.Text strong>Description</Typography.Text>}>
+                <Form.Item label={<Typography.Text strong>Description</Typography.Text>}>
                     <Typography.Paragraph>
                         An optional description to help keep track of your secret.
                     </Typography.Paragraph>
-                    <Input
-                        placeholder="The value of your secret"
-                        value={secretBuilderState.description}
-                        onChange={(event) => setDescription(event.target.value)}
-                    />
+                    <Form.Item name="description" rules={[{ whitespace: true }, { min: 1, max: 500 }]} hasFeedback>
```

Comment:
Why do we need `whitespace: true` here?

Reply:
User should not able to add only space as text, either it should be blank or some text, not just white space